wrappers/services.go

```diff
@@ -144,7 +144,7 @@ func enableServices(apps []*snap.AppInfo, inter interacter) (disable func(), err
 	var enabled []string
 	var userEnabled []string
 
-	systemSysd := systemd.New(dirs.GlobalRootDir, systemd.SystemMode, inter)
+	systemSysd := systemd.New("", systemd.SystemMode, inter)
```

Comment:
I was pondering if there should be a constant for the empty string but I think this should have different constructor instead. When talking to the host systemd use:
```
systemSysd := systemd.New(systemd.SystemMode, inter)
```
When manipulating a chroot use:
```
systemSysd := systemd.NewInChroot(...)
```

Reply:
Yes, I think it's a good idea to have a new ctor but I'd avoid "chroot" in the name as it may lead to confusion. Maybe NewForDirectory(..)?